### ISSUE TITLE
Can't click and drag panels in the timeline to change length

### ISSUE THREAD

**Issue by SpastikChuwawa** [wonderunit/storyboarder#1069]
When I try to click and drag a panel to extend it's length in the timeline, nothing happens (or it changes the length after 10 seconds have passed). I'm on Windows 10, GTX Geforce 1060, 16 GB RAM, I7 core.

**Comment by christiansass** [comment]
I experienced something similar but it was because the audio extended past the length of my boards?

**Comment by setpixel** [comment]
Yes.. this is a bug when the timeline doesnt extend beyond the window (if you have few boards). Should be straightforward to fix in the future.

**Comment by christiansass** [comment]
as a quick workaround i found adding a 1 minute board to the end so it extends past the audio allows you to scale and work with boards as intended then delete the "end board" when done
